doc/modules/grid_search.rst

```diff
@@ -12,7 +12,7 @@ Typical examples include ``C``, ``kernel`` and ``gamma`` for Support Vector
 Classifier, ``alpha`` for Lasso, etc.
```

Comment:
For instance this could be re-phrased:

```
Hyper-parameters are parameters that are not directly learnt within
estimators. In scikit-learn they are passed as arguments to the
constructor of the estimator classes. Typical examples include
``C``, ``kernel`` and ``gamma`` for Support Vector Classifier,
``alpha`` for Lasso, etc.

It is possible and recommended to search the hyper-parameter space
for the best :ref:`cross_validation` score.

...
```